sympy/stats/tests/test_discrete_rv.py

```diff
@@ -41,6 +41,13 @@ def test_Poisson():
     # issue 8248
     assert x.pspace.compute_expectation(1) == 1
 
+def test_FlorySchulz():
+    p = S.One / 5
+    x = FlorySchulz('x' , p)
+    assert E(x) == 9
+    assert variance(x) == 40
+    assert density(x)(2) - 8/125 < 0.001
```

Comment:
Please add some symbolic tests as well.

Reply:
Tests of type in L49 should be avoided. They are quite ad-hoc in nature.

Reply:
done @czgdp1807